### PR TITLE
Add AVW, KKK, and RvA

### PR DESCRIPTION
**Pull request by MaxOpperman**

### CODE REVIEW

File: module/Database/src/Database/Form/Foundation.php [hunk 1]
```diff
@@ -22,7 +22,10 @@ public function __construct(Fieldset\Meeting $meeting, Fieldset\MemberFunction $
                 'value_options' => array(
                     'committee' => 'Commissie',
                     'avc' => 'AV-Commissie',
-                    'fraternity' => 'Dispuut'
+                    'avw' => 'AV-Werkgroep',
+                    'kkk' => 'KKK',
+                    'fraternity' => 'Dispuut',
+                    'RvA' => 'RvA'
```
Comment (Koen1999) [inline module/Database/src/Database/Form/Foundation.php:28]: RvA on the left should be lowercase for consistency.

File: module/Database/view/database/index/index.phtml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <div class="jumbotron">
     <h1><span class="gewis-red">GEWIS</span> database tool</h1>
-    <p>Tijdelijke frontpage</p>
+    <p>🍑 Niet meer tijdelijke frontpage 🍑</p>
```
Comment (Koen1999) [inline module/Database/view/database/index/index.phtml:3]: Why is this change needed :sweat_smile:

**Review by Koen1999 — CHANGES_REQUESTED** [review]

File: module/Database/view/database/index/index.phtml (hunk 2 above)
Comment (MaxOpperman) [reply module/Database/view/database/index/index.phtml:3]: Because nobody else will ever make it non-temporary.

File: module/Export/src/Export/Service/Meeting.php [hunk 3]
```diff
@@ -210,7 +210,10 @@ public function getOrganId(SubDecision\Foundation $organ)
         case SubDecision\Foundation::ORGAN_TYPE_COMMITTEE:
             $type = 2;
             break;
-        case SubDecision\Foundation::ORGAN_TYPE_AV_COMMITTEE:
+        case SubDecision\Foundation::ORGAN_TYPE_AV_COMMITTEE
+            || SubDecision\Foundation::ORGAN_TYPE_KKK
+            || SubDecision\Foundation::ORGAN_TYPE_RVA
+            || SubDecision\Foundation::ORGAN_TYPE_AVW:
```
Comment (kokx) [inline module/Export/src/Export/Service/Meeting.php:216]: This is not how cases work in PHP. You should do something like this instead:

```php
case AV_COMMITTEE:
case KKK:
case AVW:
    $type = 1;
    break;
```

File: module/Export/src/Export/Service/Organ.php [hunk 4]
```diff
@@ -20,7 +20,10 @@ public function export()
             case Foundation::ORGAN_TYPE_COMMITTEE:
                 $type = 2;
                 break;
-            case Foundation::ORGAN_TYPE_AV_COMMITTEE:
+            case Foundation::ORGAN_TYPE_AV_COMMITTEE
+                || Foundation::ORGAN_TYPE_KKK
+                || Foundation::ORGAN_TYPE_RVA
+                || Foundation::ORGAN_TYPE_AVW:
```
Comment (kokx) [inline module/Export/src/Export/Service/Organ.php:26]: See other comment.

**Review by kokx — CHANGES_REQUESTED** [review]
We should also not forget that we need to add the changes in the reportdb to the website. Otherwise the website will break after the first new type of organ.

**Review by Koen1999 — APPROVED** [review]
I have tested the changes in combination with the changes to gewisweb and it works as expected.
